README.md

```diff
@@ -27,7 +27,29 @@ The resources directory contains Robot Keyword helpers for ``resincli`` and hard
 
 * Execute the following to run a example test against the qemux86-64 image you just downloaded.
 
-    ``docker run -it --rm -v <path_to_repo>:/autohat -v /dev/:/dev2 --privileged --env-file ./env.list autohat robot --exitonerror /autohat/qemux86-64.robot``
+    ``docker run -it --rm -v <path_to_repo>:/autohat -v /dev/:/dev2 --privileged --env-file ./env.list autohat robot --exitonerror /autohat/qemu.robot``
+
+#### Running with QEMU device type (Example - QEMU X86 32bit)
```

Comment:
We can remove the extra section for QEMU 32 bit - since it is the same instruction set as 64 bit?